src/components/Communicator/CommunicatorDialog/CommunicatorDialogBoardItem.component.js

```diff
@@ -288,6 +364,131 @@ class CommunicatorDialogBoardItem extends React.Component {
       this.state.imageBoard.search('/') === 0
         ? `.${this.state.imageBoard}`
         : this.state.imageBoard;
+
+    const ReportBoardDialog = () => {
+      const ReportSuccesContent = (
+        <>
+          <DialogContent>
+            <div className="CommunicatorDialog__board-report-succes">
+              <CheckCircleIcon
+                className="CommunicatorDialog__board-report-succes-icon"
+                color="primary"
+              />
+              <DialogContentText>
+                {intl.formatMessage(messages.boardReportSuccesSubtitle)}
+                <br />
+                {intl.formatMessage(messages.boardReportSuccesGratitude)}
+              </DialogContentText>
+            </div>
+          </DialogContent>
+          <DialogActions>
+            <Button
+              onClick={this.handleDialogClose.bind(this)}
+              variant="contained"
+              color="primary"
+            >
+              {intl.formatMessage(messages.boardReportClose)}
+            </Button>
+          </DialogActions>
+        </>
+      );
+
+      const ReportingContent = (
+        <>
+          <DialogContent>
+            <DialogContentText>
+              {intl.formatMessage(messages.boardInfoName)}
+              <br />
+              <br />
+              {intl.formatMessage(messages.boardReportContentSubtitle)}
+              <br />
+              <b>{board.name}</b>
```

Comment:
re-arrange the elements, see below:

![image](https://user-images.githubusercontent.com/8397930/148615930-c05d8757-d524-413f-8ea5-22e9825a44c7.png)